### ISSUE TITLE
Consulting about downloading dataset on data.public.lu

### ISSUE THREAD

**Issue by join-11223** [MISP/misp-opendata#2]
Hi, so sorry to bother you again. But I am still troubling about downloading this dataset.
For example, after clicking the links "https://data.public.lu/fr/datasets/r/d15c86a9-c907-4977-8cc0-2312c915057e" or "https://misppriv.circl.lu/attributes/restSearch/type[]:x509-fingerprint-md5/type[]:x509-fingerprint-sha1/type[]:x509-fingerprint-sha256/tags:tlp:white", it turns to page as follow:
Authentication failed. Please make sure you pass the API key of an API enabled user along in the Authorization header.
Error: The requested address '/attributes/restSearch/type[]:x509-fingerprint-md5/type[]:x509-fingerprint-sha1/type[]:x509-fingerprint-sha256/tags:tlp:white?%2Fattributes%2FrestSearch%2Ftype%5B0%5D=' was not found on this server.
Did I download it with wrong way? Or should I need to download by the API method?

**Comment by chrisr3d** [comment]
Hi,
No worries I redirected you here because it is easier to manage and keep traces of issues here.

So basically, do you know MISP and do you have access to misppriv.circl.lu?
If the answer to those questions is no, I will give you explanations and tell you what you can do next.

Anyway I will soon work on an upload functionality to post directly the JSON data in the portal, cause as you can see, currently I only post the link to the MISP platform where the data is available for people who have access.

**Comment by join-11223** [comment]
Hi,thanks for your replying. I have tried download other dataset on data.public.lu, where what I only need to do is click the “download”, and it succeed.
So, as for the MISP and the access, I think I need your help. It would be perfect if you can give me some explanations and examples.
Thank you so so so much!!!




On 07/29/2020 23:19, Christian Studer wrote:

Hi,
No worries I redirected you here because it is easier to manage and keep traces of issues here.

So basically, do you know MISP and do you have access to misppriv.circl.lu?
If the answer to those questions is no, I will give you explanations and tell you what you can do next.

Anyway I will soon work on an upload functionality to post directly the JSON data in the portal, cause as you can see, currently I only post the link to the MISP platform where the data is available for people who have access.
